### DISCUSSION TITLE
SAME ADDRESS CHANGES LOCATION NAME (AND NOTES) FOR ALL MEETINGS

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#685] [Q&A] [answered]
This seems to be a BUG that would be great to addressed.  

I have a meeting at a location that meets in a different room than the other meetings at same location. If I change the location name but keep the same address and put a note in the location section, ALL meetings at that same address also change to the same location name and the same notes. This should not be the case. 

I have 5 meetings that meet at same location, same address and same room on different days or times, and then one other meeting that meets at same location, same address, but different room.

I thought if I could rename location to something unique for the 6th meeting that I could put in location notes what was unique to that meeting (different room), but instead it changed all other 5 meetings at that address to display the new location name and location notes.

The common denominator should not be address, it should be the combination of location name and address.

I have temporarily gotten around this by adding unique notes about the location in the meeting note section.

Thank you.

**Comment by kiyote33** [comment, marked as answer]
Hi @Grateful-Viv,
In reviewing this discussion topic with the maintenance team, I learned that by design the Location Information "**Location Notes**" field should only be used for physical location information that pertains to the building (i.e. "Please enter from the parking lot on the northeast side") which would be useful information for all groups meeting at the same location. Group meeting information that is specific to an individual group (i.e. "Group meets in the main Board Room") should be noted in the Meeting Information  "**Notes**" field.  So (by design) most changes made to the Location Information will affect the display of all meetings at the same physical location.  So you're work-around is the way the software is designed to work. I hope this answers you're question.

**Reply by Grateful-Viv** [reply]
So it’s a “feature” then, not a bug.
